build/config/compiler/BUILD.gn

```diff
@@ -812,16 +813,23 @@ if (is_win) {
 
 # Default "optimization on" config. On Windows, this favors size over speed.
 config("optimize") {
+  lto_flags = []
   if (is_win) {
     # Favor size over speed, /O1 must be before the common flags. The GYP
     # build also specifies /Os and /GF but these are implied by /O1.
     cflags = [ "/O1" ] + common_optimize_on_cflags + [ "/Oi" ]
-  } else if (is_android || is_ios) {
+  } else if (is_ios) {
     cflags = [ "-Os" ] + common_optimize_on_cflags  # Favor size over speed.
```

Comment:
Using `-Oz` on iOS builds actually increases size, even though clang doc says: 

```
-Oz   Like -Os (and thus -O2), but reduces code size further.
```

Reply:
Interesting - @petrhosek might have more info.

Reply:
This is not entirely surprising. LTO usually leads to size increase due to inlining and it currently doesn't currently take `-Oz` into account. We had some discussions about this in the past, but the conclusion is that it's not quite clear what LTO with `-Oz` should do since those are to some extent conflicting options.

Reply:
Thanks for the explanation!